### DISCUSSION TITLE
Can vidstack set metadata for mediaSession?

### DISCUSSION THREAD

**Discussion by AgentSmith0** [vidstack/player#953] [Q&A] [answered]
How is it possible to set title, artist, album and artwork in vidstack for MediaMetadata (https://developer.mozilla.org/en-US/docs/Web/API/MediaMetadata/MediaMetadata)?

**Comment by mihar-22** [comment, marked as answer]
That's more an application-level concern which you can set yourself, what were you hoping the player would do here?

**Reply by AgentSmith0** [reply]
I thought it could be easier to access, because vidstack already has the artwork (poster) and the title and could automatically do that. But yeah, implementation on application-level isn't a big deal.

**Reply by mihar-22** [reply]
Hm, we can experiment setting it if it's not set... better than nothing I guess. I wonder how it looks if you just set title and a standard poster. I'll need to test it later on my phone.

**Reply by mihar-22** [reply]
Update here that in the next release we'll be automatically setting/updating the Media Session data.
